### PR TITLE
Added Leaseweb API for dns-01 verification

### PR DESCRIPTION
**Pull request by RolphH**
This is a small PR to add Leaseweb API to be used for DNS-01 verification.

### CODE REVIEW

**Comment by Neilpang** [comment]
check the dev guide first:
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by Neilpang** [comment]
add your usage here:
https://github.com/Neilpang/acme.sh/wiki/dnsapi

File: dnsapi/dns_leaseweb.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  return 1
+}
+
+####################  Private functions below ##################################
+# _acme-challenge.www.domain.com
+# returns
+# _domain=domain.com
+_get_root() {
+  domain=$1
+  i="$(echo "$fulldomain" | tr '.' ' ' | wc -w)"
```
Comment (Neilpang) [inline dnsapi/dns_leaseweb.sh]: No,  you can not just split the domain to get the root part.

See these format:
```
The full domain could be in one of the following formats:

_acme-challenge.www.example.com
_acme-challenge.example.com
_acme-challenge.example.co.uk
_acme-challenge.www.example.co.uk
_acme-challenge.sub1.sub2.www.example.co.uk
```

Use the api to get a full list of all the zones, then check one by one.

see the dev guide.

**Comment by Neilpang** [comment]
and update ?

**Comment by Neilpang** [comment]
please create a bug:
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide#10-please-create-a-new-issue-for-future-bugs

**Comment by RolphH** [comment]
Bug created

**Comment by Neilpang** [comment]
https://github.com/Neilpang/acme.sh/issues/2558

**Comment by Neilpang** [comment]
update the bug link to the usage page.

**Pull request merged**